### ISSUE TITLE
Why the world needs generalists (and specialists!)

### ISSUE THREAD

**Issue by jugglingcats** [jxyden14/nedyaj#3]
https://freakonomics.com/podcast/pima-david-epstein

(also available on Spotify)

**Comment by jugglingcats** [comment]
Hi @jxyden14 let me know what you think of this episode and if you liked it I can suggest others.

Sorry but I forgot the name of the guy who does the podcast you're listening too - an American entrepreneur of Indian heritage? Tx

**Comment by jxyden14** [comment]
Aah, Naval Ravikant

I have attached a link to one of his podcasts here:
https://open.spotify.com/show/7qZAVw03FuurfYnWIWwkHY

**Comment by jugglingcats** [comment]
Thanks Jayden just noticed this - will have a listen! Did you try the freakonomics podcast yet?

**Comment by jugglingcats** [comment]
> Aah, Naval Ravikant I have attached a link to one of his podcasts here: https://open.spotify.com/show/7qZAVw03FuurfYnWIWwkHY
> […](#)


Listened to one of these today - really excellent! He and a guest were talking about one of my favourite books.

**Comment by jxyden14** [comment]
Yes Sir I have got a start on it, sounds interesting at first glance... :)

**Comment by jugglingcats** [comment]
@jxyden14 here is the Spotify link: https://open.spotify.com/episode/0p49QUitbwRG0jgUC4BWCv?si=d65b425c710448d4

**Comment by jugglingcats** [comment]
Hi @jxyden14 feel free to close this issue if you listened to the episode
